tests/integration/acp/dac/add_policy/with_no_perms_test.go

```diff
@@ -63,25 +52,15 @@ func TestACP_AddPolicy_NoPermissionsMultiRelations_ValidID(t *testing.T) {
 				Identity: testUtils.ClientIdentity(1),
 
 				Policy: `
-                    name: test
-                    description: a policy
-
-                    actor:
-                      name: actor
-
-                    resources:
-                      users:
-                        permissions:
-
-                        relations:
-                          owner:
-                            types:
-                              - actor
-                          reader:
-                            types:
-                              - actor
-
-                `,
+description: a policy
+name: test
+resources:
+- name: users
+  relations:
+  - name: reader
+    types:
+    - actor
```

Comment:
todo: 2 problems:

1) This test does not have multi relations now as owner is gone, so perhaps make it a multi relations by introducing one
2) Is again a duplicate of `TestACP_AddPolicy_NoPermissionsLabelMultiRelations_ValidID` because it should have the explicit label `permissions:` like:
```suggestion
description: a policy
name: test
resources:
- name: users
  permissions:
  relations:
  - name: reader
    types:
    - actor
```